File: src/Backend/Domain/Auth.Entity/Ids4Entity/PersistedGrant.cs
using System;
using Dapper.Contrib.Plus;

namespace Auth.Entity.Ids4Entity
{
    /// <summary>
    /// PersistedGrants
    /// </summary>
    [Table("persisted_grants")]
    public class PersistedGrant
    {
        /// <summary>
        /// Key
        /// </summary>
        public string Key { get; set; }

        /// <summary>
        /// Type
        /// </summary>
        public string Type { get; set; }

        /// <summary>
        /// SubjectId
        /// </summary>
        public string SubjectId { get; set; }

        /// <summary>
        /// SessionId
        /// </summary>
        public string SessionId { get; set; }

        /// <summary>
        /// ClientId
        /// </summary>
        public string ClientId { get; set; }

        /// <summary>
        /// Description
        /// </summary>
        public string Description { get; set; }

        /// <summary>
        /// CreationTime
        /// </summary>
        public string CreationTime { get; set; }

        /// <summary>
        /// Expiration
        /// </summary>
        public string Expiration { get; set; }

        /// <summary>
        /// ConsumedTime
        /// </summary>
        public string ConsumedTime { get; set; }

        /// <summary>
        /// Data
        /// </summary>
        public string Data { get; set; }
    }
}

File: src/FrontEnd/auth-admin/src/types/entities/ids/persisted-grant.ts
interface IPersistedGrant {
  key: string;
  type: string;
  subjectId: string;
  sessionId: string;
  clientId: string;
  description: string;
  creationTime: string;
  expiration: string;
  consumedTime: string;
  data: string;
}

export default class PersistedGrant implements IPersistedGrant {
  key: string = '';
  type: string = '';
  subjectId: string = '';
  sessionId: string = '';
  clientId: string = '';
  description: string = '';
  creationTime: string = '';
  expiration: string = '';
  consumedTime: string = '';
  data: string = '';
}